# Notebook 22  
# SQL Healthcare Operations Analytics

---

# Purpose

This notebook analyzes:

- Resource utilization
- Healthcare operations
- Financial burden
- Cost drivers
- High-utilization populations
- High-cost populations

The goal is to understand:

> Which patient populations consume healthcare resources and drive operational burden?

---

# Input Table

Main table used:

healthcare_catalog.gold.population_health_dashboard

Contains:

- demographics
- risk
- utilization
- claims
- disease burden
- medication burden
- costs

---

# Expected Outputs

We will create:

✓ Utilization KPIs

✓ Operational KPIs

✓ Financial KPIs

✓ Dashboard-ready metrics

✓ Cost driver insights

---

# Sections

SECTION 1 → Resource Utilization Analytics

SECTION 2 → Healthcare Operations Analytics

SECTION 3 → Financial Analytics

SECTION 4 → Cost Driver Analytics

SECTION 5 → Comparative & Ranking Analytics

# ==================================================

# SECTION 1
# Resource Utilization Analytics

Status:

STARTED

# ==================================================

## Goal

Understand how healthcare resources are consumed.

Examples:

- encounters
- inpatient burden
- ambulatory burden
- high utilizers
- frequent users

---

## Business Questions

1. Do high-risk patients consume more healthcare resources?

2. Do seniors utilize healthcare more?

3. Does chronic disease increase utilization?

4. Who are high utilizers?

---

## Expected Outputs

KPIs:

- average encounters
- encounter burden
- utilization patterns
- high-utilizer groups

Dashboard metrics:

- utilization by risk
- utilization by age

# Debug Step

Before performing utilization analytics, we need to inspect the columns available in:

healthcare_catalog.gold.population_health_dashboard

This helps verify:

- risk variables
- utilization variables
- claim variables
- demographic variables

Always inspect schema before analytics.

In [0]:
%sql
DESCRIBE healthcare_catalog.gold.population_health_dashboard;

col_name,data_type,comment
patient_id,string,null
gender,string,null
age,bigint,null
city,string,null
state,string,null
marital_status,string,null
total_encounters,bigint,null
ambulatory_encounters,bigint,null
emergency_encounters,bigint,null
inpatient_encounters,bigint,null


# Query 1.1

# Population Health Risk Category vs Resource Utilization

---

## Question

Do higher-risk patients consume more healthcare encounters?

---

## What we are doing

We are grouping patients by:

population_health_risk_category

Then we calculate:

- total patients
- average encounters
- minimum encounters
- maximum encounters
- average emergency encounters
- average inpatient encounters

---

## Why we are doing it

We want to see whether higher-risk populations create more resource utilization.

In healthcare operations:

Higher Risk

↓

More Encounters

↓

More Emergency / Inpatient Use

↓

Higher Operational Burden

---

## How this works

We use:

GROUP BY

AVG()

MIN()

MAX()

COUNT()

to summarize utilization by risk category.

---

## Expected Output

Each row should show one risk category.

Example:

| population_health_risk_category | total_patients | avg_total_encounters |
|---|---:|---:|
| High Risk | xx | xx |
| Medium Risk | xx | xx |
| Low Risk | xx | xx |

---

## Interview Relevance

This is a common healthcare analytics question:

"How do you identify which patient populations consume the most resources?"

In [0]:
%sql
SELECT

population_health_risk_category,

COUNT(*) AS total_patients,

ROUND(AVG(total_encounters), 2)
    AS avg_total_encounters,

MIN(total_encounters)
    AS min_total_encounters,

MAX(total_encounters)
    AS max_total_encounters,

ROUND(AVG(emergency_encounters), 2)
    AS avg_emergency_encounters,

ROUND(AVG(inpatient_encounters), 2)
    AS avg_inpatient_encounters

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY population_health_risk_category

ORDER BY avg_total_encounters DESC;

population_health_risk_category,total_patients,avg_total_encounters,min_total_encounters,max_total_encounters,avg_emergency_encounters,avg_inpatient_encounters
High Risk,77,122.0,24,1563,5.86,6.38
Medium Risk,231,53.13,14,140,1.7,0.84
Low Risk,247,24.87,2,106,0.73,0.05


# Query 1.1 Results

# Population Health Risk vs Resource Utilization

---

## Findings

| Risk Group | Avg Encounters | Avg Emergency | Avg Inpatient |
|------------|----------------|----------------|----------------|
| High Risk | 122 | 5.86 | 6.38 |
| Medium Risk | 53.13 | 1.70 | 0.84 |
| Low Risk | 24.87 | 0.73 | 0.05 |

---

## Major Insight 1

High-risk populations consume dramatically more healthcare resources.

Comparison:

High Risk:

122 average encounters

Low Risk:

24.87 average encounters

Approximate difference:

High-risk patients use ~5x more encounters.

---

## Major Insight 2

Emergency utilization increases with risk.

Average emergency encounters:

Low Risk → 0.73

High Risk → 5.86

High-risk populations show significantly greater emergency dependency.

---

## Major Insight 3

Inpatient burden is concentrated among high-risk populations.

Average inpatient encounters:

Low Risk → 0.05

High Risk → 6.38

This suggests:

Higher risk

↓

More admissions

↓

Greater operational burden

↓

Higher healthcare cost

---

## Business Interpretation

High-risk populations likely drive:

- hospital workload
- admission burden
- staffing requirements
- care management demand
- healthcare spending

---

## Possible Intervention Programs

Healthcare systems may target:

✓ Chronic disease management

✓ Preventive outreach

✓ Care coordination

✓ Population health programs

to reduce utilization burden.

---

## Executive Summary

Small high-risk populations create disproportionate healthcare resource utilization.

--------------------------------------------------

SECTION 1:
Resource Utilization Analytics

Completed:

✓ Query 1.1
Risk vs Utilization

Status:

IN PROGRESS

Next:

Query 1.2
Age Group vs Resource Utilization

--------------------------------------------------

# Query 1.2

# Age Group vs Resource Utilization

---

## Question

Do older populations consume more healthcare resources?

---

## What we are doing

We will create age groups:

Pediatric

Adult

Senior

Then compare:

- average encounters
- emergency encounters
- inpatient encounters

---

## Why we are doing it

Healthcare systems often observe:

Older populations

↓

More chronic disease

↓

Higher utilization

↓

Higher spending

We want to verify whether this exists in our data.

---

## Expected Output

Each row:

One age group

Columns:

Average utilization metrics

---

## Interview Relevance

Common healthcare analytics question:

"Which age populations consume most healthcare services?"

In [0]:
%sql
SELECT

CASE

WHEN age < 18
THEN 'Pediatric'

WHEN age BETWEEN 18 AND 64
THEN 'Adult'

ELSE 'Senior'

END AS age_group,

COUNT(*) AS total_patients,

ROUND(AVG(total_encounters),2)
AS avg_total_encounters,

ROUND(AVG(emergency_encounters),2)
AS avg_emergency_encounters,

ROUND(AVG(inpatient_encounters),2)
AS avg_inpatient_encounters

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY age_group

ORDER BY avg_total_encounters DESC;

age_group,total_patients,avg_total_encounters,avg_emergency_encounters,avg_inpatient_encounters
Senior,136,88.84,3.13,2.94
Adult,339,42.1,1.61,0.87
Pediatric,80,18.24,0.68,0.01


# Query 1.2 Results

# Age Group vs Resource Utilization

---

## Findings

| Age Group | Avg Encounters | Avg Emergency | Avg Inpatient |
|-----------|----------------|----------------|----------------|
| Senior | 88.84 | 3.13 | 2.94 |
| Adult | 42.10 | 1.61 | 0.87 |
| Pediatric | 18.24 | 0.68 | 0.01 |

---

## Major Insight 1

Healthcare utilization increases with age.

Average encounters:

Pediatric → 18.24

Adult → 42.10

Senior → 88.84

This shows:

Older populations consume significantly more healthcare resources.

---

## Major Insight 2

Emergency utilization rises with age.

Emergency encounters:

Senior → 3.13

Adult → 1.61

Pediatric → 0.68

Seniors depend more heavily on urgent healthcare services.

---

## Major Insight 3

Inpatient burden is concentrated among seniors.

Average inpatient encounters:

Senior → 2.94

Adult → 0.87

Pediatric → 0.01

This suggests:

Older populations create greater hospital admission burden.

---

## Business Interpretation

Senior populations may drive:

- hospital utilization
- admission burden
- care coordination needs
- chronic disease workload
- operational costs

---

## Executive Summary

Senior populations consume disproportionately high healthcare resources and may require targeted preventive programs.

--------------------------------------------------

SECTION 1:
Resource Utilization Analytics

Completed:

✓ Query 1.1
Risk vs Utilization

✓ Query 1.2
Age Group vs Utilization

Status:

IN PROGRESS

Next:

Query 1.3
Chronic Disease Burden vs Resource Utilization

--------------------------------------------------

# Query 1.3

# Chronic Disease Burden vs Resource Utilization

---

## Question

Do patients with more chronic diseases consume more healthcare resources?

---

## What we are doing

We will group patients by:

chronic_disease_count

Then compare:

- average encounters
- emergency encounters
- inpatient encounters

---

## Why we are doing it

Chronic diseases often increase:

Medication burden

↓

Hospital visits

↓

Emergency encounters

↓

Admissions

↓

Healthcare utilization

We want to measure whether utilization rises as disease burden increases.

---

## Expected Output

Each row:

One chronic disease count

Metrics:

Average utilization

---

## Interview Relevance

Common healthcare question:

"How does multimorbidity affect healthcare utilization?"

# Query 1.3

# Chronic Disease Burden vs Resource Utilization

---

## Question

Do patients with more chronic diseases consume more healthcare resources?

---

## What we are doing

We will group patients by:

chronic_disease_count

Then compare:

- average encounters
- emergency encounters
- inpatient encounters

---

## Why we are doing it

Chronic diseases often increase:

Medication burden

↓

Hospital visits

↓

Emergency encounters

↓

Admissions

↓

Healthcare utilization

We want to measure whether utilization rises as disease burden increases.

---

## Expected Output

Each row:

One chronic disease count

Metrics:

Average utilization

---

## Interview Relevance

Common healthcare question:

"How does multimorbidity affect healthcare utilization?"

In [0]:
%sql
SELECT

chronic_disease_count,

COUNT(*) AS total_patients,

ROUND(AVG(total_encounters),2)
AS avg_total_encounters,

ROUND(AVG(emergency_encounters),2)
AS avg_emergency_encounters,

ROUND(AVG(inpatient_encounters),2)
AS avg_inpatient_encounters

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY chronic_disease_count

ORDER BY chronic_disease_count;

chronic_disease_count,total_patients,avg_total_encounters,avg_emergency_encounters,avg_inpatient_encounters
0,114,21.54,0.86,0.01
1,90,36.46,1.71,0.5
2,167,45.1,1.57,0.68
3,115,54.29,1.8,1.88
4,56,106.05,2.59,4.23
5,11,139.27,11.55,3.91
6,2,415.5,15.5,20.5


# Query 1.3 Results

# Chronic Disease Burden vs Resource Utilization

---

## Findings

| Chronic Diseases | Avg Encounters | Avg Emergency | Avg Inpatient |
|------------------|----------------|----------------|----------------|
| 0 | 21.54 | 0.86 | 0.01 |
| 1 | 36.46 | 1.71 | 0.50 |
| 2 | 45.10 | 1.57 | 0.68 |
| 3 | 54.29 | 1.80 | 1.88 |
| 4 | 106.05 | 2.59 | 4.23 |
| 5 | 139.27 | 11.55 | 3.91 |
| 6 | 415.50 | 15.50 | 20.50 |

---

## Major Insight 1

Healthcare utilization increases as chronic disease burden increases.

Average encounters:

0 chronic diseases → 21.54

6 chronic diseases → 415.50

This suggests:

Patients with multiple chronic conditions consume dramatically more healthcare resources.

---

## Major Insight 2

Emergency utilization rises sharply with disease burden.

Emergency encounters:

0 diseases → 0.86

5 diseases → 11.55

6 diseases → 15.50

Patients with multimorbidity rely heavily on emergency care.

---

## Major Insight 3

Hospital admissions increase substantially with chronic disease burden.

Average inpatient encounters:

0 diseases → 0.01

6 diseases → 20.50

Multimorbidity appears strongly associated with hospitalization burden.

---

## Business Interpretation

Patients with multiple chronic diseases may drive:

- admission burden
- emergency utilization
- staffing requirements
- operational workload
- healthcare spending

---

## Population Health Interpretation

Multimorbidity populations may benefit from:

✓ chronic disease management

✓ care coordination

✓ preventive programs

✓ medication management

✓ population health interventions

---

## Executive Summary

Small multimorbidity populations generate disproportionate healthcare resource utilization.

--------------------------------------------------

SECTION 1:
Resource Utilization Analytics

Completed:

✓ Query 1.1
Risk vs Utilization

✓ Query 1.2
Age vs Utilization

✓ Query 1.3
Chronic Disease Burden vs Utilization

Status:

IN PROGRESS

Next:

Query 1.4
Identify High Utilizers

--------------------------------------------------

# Query 1.4

# Identify High Utilizer Patients

---

## Question

Who are the highest healthcare utilizers?

---

## What we are doing

We will rank patients by:

total_encounters

and identify the top utilizers.

---

## Why we are doing it

A small group of patients often generates:

Most encounters

↓

Most admissions

↓

Most cost

↓

Most operational burden

Healthcare systems call these:

"High Utilizers"

---

## Expected Output

Top patients with:

- highest encounters
- chronic disease burden
- cost burden
- risk level

---

## Interview Relevance

Common healthcare operations question:

"How do you identify frequent healthcare users?"

In [0]:
%sql
SELECT

patient_id,

population_health_risk_category,

age,

chronic_disease_count,

total_encounters,

emergency_encounters,

inpatient_encounters,

total_claim_cost

FROM healthcare_catalog.gold.population_health_dashboard

ORDER BY total_encounters DESC

LIMIT 10;

patient_id,population_health_risk_category,age,chronic_disease_count,total_encounters,emergency_encounters,inpatient_encounters,total_claim_cost
ef167059-cef0-12c4-49db-993ca3a20c01,High Risk,97,4,1563,5,17,2190959.000000004
e23c7a6e-6ef2-3932-1361-b73b4c8a3961,High Risk,111,6,705,29,0,1.3535234989999948E7
8c6ae452-5f8c-9ff6-006d-c6c860acf5cd,High Risk,108,5,638,115,0,345415.1699999947
55c5b8d3-99d0-58ad-8444-e42bb81bd5c7,High Risk,84,4,412,23,0,1.0331274649999976E7
eb7cffc3-28d2-e8b0-72bc-8e86860ed313,High Risk,64,4,182,6,57,370014.6099999982
3dfce025-b696-711a-8c2d-6903736e2882,High Risk,63,4,164,5,0,244556.02999999892
7bc66758-5d59-0b38-1532-29a33fa146c1,High Risk,114,3,158,6,5,1064065.9899999993
8bf6aa92-645f-3c82-ddcd-5851496a6aa8,High Risk,59,4,145,3,59,396557.4699999996
a8c08d9f-a2ae-46c6-0371-f596ed13d1cf,Medium Risk,99,5,140,1,1,835004.2299999997
72f91cc7-fb63-84e8-32ff-86456bfa446e,High Risk,83,3,139,5,6,174786.89999999973


# Query 1.4 Results

# High Utilizer Patient Analysis

---

## Findings

Top healthcare utilizers show:

Extremely high:

- encounter burden
- chronic disease burden
- claim cost
- advanced age
- risk level

---

## Major Insight 1

Most high utilizers belong to:

High Risk population

This suggests:

Clinical risk strongly predicts utilization burden.

---

## Major Insight 2

High utilizers frequently have:

Multiple chronic diseases

Examples:

4 diseases

5 diseases

6 diseases

This indicates multimorbidity may drive healthcare consumption.

---

## Major Insight 3

Utilization and spending appear connected.

Examples:

705 encounters

↓

$13.5M claim cost

1563 encounters

↓

$2.19M claim cost

High utilization may create significant financial burden.

---

## Major Insight 4

Older populations dominate highest utilization rankings.

Many high utilizers:

Age >80

This supports previous findings:

Senior populations consume disproportionate healthcare resources.

---

## Business Interpretation

These patients may require:

✓ intensive care management

✓ chronic disease programs

✓ preventive interventions

✓ case management

✓ utilization reduction strategies

---

## Healthcare Operations Interpretation

A small subset of patients generates:

- large operational workload
- large resource burden
- high spending

This follows the:

80/20 utilization principle

Small populations drive most healthcare utilization.

---

## Executive Summary

High-risk, older, multimorbidity patients dominate utilization burden and likely drive operational cost.

--------------------------------------------------

SECTION COMPLETE

SECTION:

Resource Utilization Analytics

Completed Analyses:

✓ Risk vs Utilization

✓ Age vs Utilization

✓ Chronic Disease Burden vs Utilization

✓ High Utilizer Identification

---

Main Findings

1.

High-risk populations use ~5x more encounters.

2.

Senior populations consume substantially more resources.

3.

Multimorbidity strongly increases utilization.

4.

Small high-risk populations generate disproportionate workload.

---

Business Impact

Supports:

- population health programs
- utilization reduction
- care coordination
- operations planning

---

Outputs Generated

Resource utilization KPIs

High-utilizer insights

Operational burden metrics

---

Status:

SECTION 1 COMPLETE

Proceeding to:

SECTION 2

Healthcare Operations Analytics

--------------------------------------------------

# SECTION 2

# Healthcare Operations Analytics

Status:

STARTED

---

# Objective

Evaluate operational burden within healthcare systems.

This section focuses on:

- Emergency department burden
- Inpatient burden
- Hospital workload
- Utilization concentration
- Care delivery pressure

The purpose is to identify:

> Which patient populations create the greatest operational demand?

---

# Why This Matters

Healthcare operations teams monitor:

- emergency utilization
- admission rates
- inpatient burden
- resource allocation
- staffing needs

Understanding these patterns supports:

✓ Hospital capacity planning

✓ Care coordination

✓ Workforce planning

✓ Utilization management

✓ Operational efficiency

---

# Expected Outputs

Operational KPIs:

- emergency burden
- admission burden
- inpatient utilization
- operational workload concentration

Dashboard-ready metrics:

- burden by risk
- burden by age
- burden by disease complexity

# Query 2.1

# Emergency Utilization by Risk Category

---

## Business Question

Which populations generate the greatest emergency department burden?

---

## What We Are Doing

Compare:

population_health_risk_category

against:

- average emergency encounters
- maximum emergency encounters

---

## Why We Are Doing It

Emergency utilization often indicates:

Higher clinical severity

↓

Higher operational pressure

↓

Higher staffing demand

↓

Higher healthcare cost

---

## How This Works

We summarize emergency burden using:

COUNT()

AVG()

MAX()

GROUP BY

---

## Expected Output

Risk categories ranked by:

Average emergency utilization

Possible finding:

High-risk populations create disproportionate emergency burden.

---

## Business Value

This analysis helps healthcare systems identify populations requiring:

- preventive intervention
- chronic disease programs
- emergency reduction strategies

In [0]:
%sql
SELECT

population_health_risk_category,

COUNT(*) AS total_patients,

ROUND(
AVG(emergency_encounters),2
) AS avg_emergency_encounters,

MAX(emergency_encounters)
AS max_emergency_encounters,

ROUND(
AVG(inpatient_encounters),2
) AS avg_inpatient_encounters

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY population_health_risk_category

ORDER BY avg_emergency_encounters DESC;

population_health_risk_category,total_patients,avg_emergency_encounters,max_emergency_encounters,avg_inpatient_encounters
High Risk,77,5.86,115,6.38
Medium Risk,231,1.7,9,0.84
Low Risk,247,0.73,3,0.05


# Query 2.1 Results

# Emergency Department Burden by Risk Category

---

## Findings

| Risk Category | Avg Emergency | Max Emergency | Avg Inpatient |
|---------------|---------------:|---------------:|---------------:|
| High Risk | 5.86 | 115 | 6.38 |
| Medium Risk | 1.70 | 9 | 0.84 |
| Low Risk | 0.73 | 3 | 0.05 |

---

# Major Findings

## Finding 1

High-risk populations create substantially greater emergency burden.

Average emergency encounters:

Low Risk → 0.73

High Risk → 5.86

Approximate increase:

High Risk ≈ 8x greater emergency utilization

---

## Finding 2

Maximum emergency burden is highly concentrated.

Observed:

High Risk

Maximum emergency encounters:

115

This indicates presence of:

Super-utilizers

or

Complex care populations

---

## Finding 3

Emergency utilization and inpatient utilization rise together.

High Risk:

Emergency = 5.86

Inpatient = 6.38

This suggests:

Increasing risk burden

↓

Emergency burden

↓

Admission burden

↓

Operational pressure

---

# Operational Interpretation

High-risk populations may drive:

- emergency department workload
- staffing demand
- admission pressure
- bed occupancy
- hospital resource utilization

---

# Executive Summary

Healthcare operational burden is disproportionately concentrated among high-risk populations.

# Query 2.2

# Admission Burden by Age Group

---

## Business Question

Which age populations generate the greatest inpatient burden?

---

## What We Are Doing

Create:

Pediatric

Adult

Senior

Then compare:

Average inpatient encounters

Average encounter duration

---

## Why

Admissions often indicate:

Higher acuity

↓

Greater operational demand

↓

Longer care episodes

↓

Higher hospital cost

---

## Expected Output

Age groups ranked by:

- inpatient utilization
- encounter duration

---

## Business Value

Supports:

Hospital capacity planning

Aging population analytics

Operational forecasting

In [0]:
%sql
SELECT

CASE

WHEN age < 18
THEN 'Pediatric'

WHEN age BETWEEN 18 AND 64
THEN 'Adult'

ELSE 'Senior'

END AS age_group,

COUNT(*) AS total_patients,

ROUND(
AVG(inpatient_encounters),2
) AS avg_inpatient_encounters,

ROUND(
AVG(avg_encounter_duration_hours),2
) AS avg_encounter_duration_hours

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY age_group

ORDER BY avg_inpatient_encounters DESC;

age_group,total_patients,avg_inpatient_encounters,avg_encounter_duration_hours
Senior,136,2.94,2.9
Adult,339,0.87,1.92
Pediatric,80,0.01,0.33


# Query 2.3

# Operational Workload by Chronic Disease Burden

---

## Business Question

Do patients with multiple chronic diseases generate greater operational workload?

---

## What We Are Doing

Group patients by:

chronic_disease_count

Then compare:

- inpatient encounters
- encounter duration
- emergency encounters

---

## Why

Multimorbidity often leads to:

Complex care

↓

More admissions

↓

Longer treatment

↓

Greater operational burden

---

## Expected Output

Increasing chronic disease burden

↓

Increasing workload metrics

---

## Business Value

Supports:

Population health planning

Care management

Operational forecasting

In [0]:
%sql
SELECT

chronic_disease_count,

COUNT(*) AS total_patients,

ROUND(
AVG(emergency_encounters),2
) AS avg_emergency,

ROUND(
AVG(inpatient_encounters),2
) AS avg_inpatient,

ROUND(
AVG(avg_encounter_duration_hours),2
) AS avg_duration_hours

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY chronic_disease_count

ORDER BY chronic_disease_count;

chronic_disease_count,total_patients,avg_emergency,avg_inpatient,avg_duration_hours
0,114,0.86,0.01,0.39
1,90,1.71,0.5,0.98
2,167,1.57,0.68,3.09
3,115,1.8,1.88,1.93
4,56,2.59,4.23,2.38
5,11,11.55,3.91,5.45
6,2,15.5,20.5,4.26


# Query 2.3 Results

# Operational Workload by Chronic Disease Burden

---

## Findings

| Chronic Diseases | Avg Emergency | Avg Inpatient | Avg Duration (hrs) |
|-----------------|---------------:|---------------:|--------------------:|
| 0 | 0.86 | 0.01 | 0.39 |
| 1 | 1.71 | 0.50 | 0.98 |
| 2 | 1.57 | 0.68 | 3.09 |
| 3 | 1.80 | 1.88 | 1.93 |
| 4 | 2.59 | 4.23 | 2.38 |
| 5 | 11.55 | 3.91 | 5.45 |
| 6 | 15.50 | 20.50 | 4.26 |

---

# Major Findings

## Finding 1

Emergency utilization increases substantially with multimorbidity.

Emergency encounters:

0 diseases → 0.86

6 diseases → 15.50

Observed:

Patients with multiple chronic conditions create greater emergency burden.

---

## Finding 2

Admission burden rises sharply with disease complexity.

Average inpatient encounters:

0 diseases → 0.01

6 diseases → 20.50

This indicates:

Complex patients drive hospital admissions.

---

## Finding 3

Encounter duration generally increases with disease burden.

Patients with:

5 chronic diseases

Average encounter duration:

5.45 hours

Longer care episodes increase:

- staffing requirements
- care coordination effort
- operational workload

---

# Operational Interpretation

Multimorbidity populations may disproportionately drive:

- emergency workload
- inpatient demand
- care complexity
- hospital resource consumption

---

# Executive Summary

Patients with multiple chronic diseases generate significantly higher operational burden and may require targeted care coordination strategies.

# Query 2.4

# Identify Highest Operational Burden Patients

---

## Business Question

Which patients create the greatest operational workload?

---

## What We Are Doing

Rank patients by:

- inpatient encounters
- emergency encounters
- encounter duration

---

## Why

Healthcare systems often identify:

High operational burden patients

These populations may require:

Case management

Care coordination

Special programs

---

## Expected Output

Top patients generating:

Highest operational workload

In [0]:
%sql
SELECT

patient_id,

population_health_risk_category,

age,

chronic_disease_count,

emergency_encounters,

inpatient_encounters,

ROUND(
avg_encounter_duration_hours,2
) AS avg_duration_hours,

total_claim_cost

FROM healthcare_catalog.gold.population_health_dashboard

ORDER BY

inpatient_encounters DESC,
emergency_encounters DESC

LIMIT 10;

patient_id,population_health_risk_category,age,chronic_disease_count,emergency_encounters,inpatient_encounters,avg_duration_hours,total_claim_cost
c03807d7-0b42-0490-23f0-8caeb7c38e74,High Risk,99,4,4,67,14.85,1597934.100000001
8bf6aa92-645f-3c82-ddcd-5851496a6aa8,High Risk,59,4,3,59,19.2,396557.4699999996
eb7cffc3-28d2-e8b0-72bc-8e86860ed313,High Risk,64,4,6,57,51.81,370014.6099999982
10a1709a-5dde-1e3a-4c06-d8de03764712,Medium Risk,68,3,2,48,14.54,1211655.7099999997
96df4283-1388-506c-e88f-77cd4232e1b4,Medium Risk,83,3,2,42,14.76,1023886.0900000002
ed803173-5e55-aecd-5055-591164ee3005,High Risk,70,6,2,41,8.23,352239.2300000001
03e502b6-b810-06c1-7d65-83db077ed3ee,High Risk,58,3,3,40,9.1,969762.9900000009
5b2f8ddd-408a-884c-00e9-047b13e6cc45,High Risk,26,2,2,34,14.96,182016.33000000005
9cbf47d8-b59a-ef34-c330-77accc084c17,High Risk,91,5,1,24,50.74,172515.77999999988
da97668c-bf70-6c4d-3157-24aae70ed237,Medium Risk,97,1,2,23,4.36,1084982.6500000006


# Query 2.4 Results

# Highest Operational Burden Patients

---

## Findings

Top operational burden patients show:

- elevated inpatient encounters
- long encounter duration
- advanced age
- multiple chronic diseases
- high claims burden

---

# Major Findings

## Finding 1

Operational workload is concentrated among high-risk populations.

Most highest-burden patients belong to:

High Risk category

Observed:

High risk patients dominate inpatient burden rankings.

---

## Finding 2

Long encounter durations increase workload intensity.

Examples:

Patient duration:

51.81 hours

50.74 hours

19.20 hours

Long care episodes require:

- staffing
- monitoring
- inpatient resources
- care coordination

---

## Finding 3

Multimorbidity contributes to operational burden.

High burden patients frequently show:

4 chronic diseases

5 chronic diseases

6 chronic diseases

Complex patients increase care workload.

---

## Finding 4

Operational burden appears linked with financial burden.

Examples:

Patients with high admissions often exhibit:

>$1M total claim cost

This suggests:

Operational burden

↓

Financial burden

---

# Operational Interpretation

Highest burden populations may require:

✓ care coordination

✓ chronic disease programs

✓ intensive case management

✓ utilization reduction initiatives

---

# Executive Summary

Healthcare workload is disproportionately generated by older, multimorbidity, high-risk populations with prolonged encounters and elevated admission burden.

--------------------------------------------------

SECTION COMPLETE

SECTION:

Healthcare Operations Analytics

Completed Analyses:

✓ Emergency Burden by Risk

✓ Admission Burden by Age

✓ Operational Workload by Chronic Disease Burden

✓ Highest Operational Burden Patients

---

# Main Findings

1.

High-risk populations generate greatest emergency burden.

2.

Senior populations drive admission burden.

3.

Multimorbidity substantially increases workload.

4.

Operational burden is concentrated among small high-risk populations.

---

# Operational Impact

Supports:

- hospital capacity planning
- staffing decisions
- utilization management
- care coordination

---

# Outputs Generated

Emergency burden KPIs

Admission burden KPIs

Operational workload metrics

High-burden patient identification

---

Status:

SECTION 2 COMPLETE

Proceeding to:

SECTION 3

Financial Analytics

--------------------------------------------------

# SECTION 3

# Financial Analytics

Status:

STARTED

---

# Objective

Evaluate healthcare spending patterns and identify:

- high-cost populations
- financial burden drivers
- cost concentration
- populations generating elevated claims burden

This section answers:

> Which patient populations drive healthcare spending?

---

# Why This Matters

Healthcare financial analytics supports:

✓ Cost containment

✓ Population health programs

✓ Payer analytics

✓ Resource allocation

✓ Value-based care

✓ Risk management

---

# Expected Outputs

Financial KPIs:

- cost by risk
- cost by age
- cost by multimorbidity
- high-cost populations

Dashboard-ready metrics:

- spending concentration
- cost burden distribution

# Query 3.1

# Claim Cost by Population Health Risk Category

---

## Business Question

Do high-risk populations generate greater healthcare spending?

---

## What We Are Doing

Compare:

population_health_risk_category

against:

- average claim cost
- maximum claim cost
- total claim cost

---

## Why We Are Doing It

Higher clinical risk often leads to:

More encounters

↓

More admissions

↓

More treatment

↓

Higher spending

We want to measure whether financial burden rises with risk.

---

## How This Works

We summarize:

AVG()

SUM()

MAX()

GROUP BY

---

## Expected Output

Risk groups ranked by spending.

Possible pattern:

High Risk

↓

Highest claims burden

---

## Business Value

Supports:

- cost reduction initiatives
- payer analytics
- risk stratification
- financial planning

In [0]:
%sql
SELECT

population_health_risk_category,

COUNT(*) AS total_patients,

ROUND(
AVG(total_claim_cost),2
) AS avg_total_claim_cost,

ROUND(
SUM(total_claim_cost),2
) AS total_claim_cost_population,

ROUND(
MAX(total_claim_cost),2
) AS max_total_claim_cost

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY population_health_risk_category

ORDER BY avg_total_claim_cost DESC;

population_health_risk_category,total_patients,avg_total_claim_cost,total_claim_cost_population,max_total_claim_cost
High Risk,77,748090.65,5.760297976E7,1.353523499E7
Medium Risk,231,270689.56,6.252928741E7,1427488.81
Low Risk,247,54265.35,1.340354225E7,905423.32


# Query 3.1 Results

# Claim Cost by Population Health Risk Category

---

## Findings

| Risk Category | Avg Claim Cost | Total Population Cost | Max Claim Cost |
|---------------|----------------:|-----------------------:|----------------:|
| High Risk | $748,090 | $57.6M | $13.5M |
| Medium Risk | $270,690 | $62.5M | $1.43M |
| Low Risk | $54,265 | $13.4M | $905K |

---

# Major Findings

## Finding 1

Average spending increases sharply with risk.

Average cost:

Low Risk → $54K

High Risk → $748K

Observed:

High-risk patients cost ~14x more than low-risk patients.

---

## Finding 2

Medium-risk populations generate highest aggregate spending.

Total population spending:

Medium Risk → $62.5M

High Risk → $57.6M

Reason:

Larger population size

This indicates:

Moderate-risk populations may create substantial cumulative financial burden.

---

## Finding 3

Extreme cost concentration exists among high-risk populations.

Maximum claim cost:

High Risk → $13.5M

Observed:

Small populations generate extreme spending.

---

# Financial Interpretation

Healthcare spending appears concentrated among:

- high-risk populations
- moderate-risk populations with large volume
- complex patients

---

# Executive Summary

Clinical risk is strongly associated with elevated financial burden, while medium-risk populations generate substantial aggregate spending through population size.

--------------------------------------------------

SECTION 3:
Financial Analytics

Completed:

✓ Query 3.1
Cost by Risk Category

Status:

IN PROGRESS

Next:

Query 3.2

Claim Cost by Age Group

--------------------------------------------------

# Query 3.2

# Claim Cost by Age Group

---

## Business Question

Do older populations generate greater healthcare spending?

---

## What We Are Doing

Segment patients into:

Pediatric

Adult

Senior

Then compare:

- average claim cost
- total claim cost

---

## Why

Older populations often exhibit:

More chronic disease

↓

Higher utilization

↓

Greater treatment burden

↓

Higher cost

---

## Expected Output

Age groups ranked by spending.

---

## Business Value

Supports:

- aging population planning
- healthcare forecasting
- financial planning

In [0]:
%sql
SELECT

CASE

WHEN age < 18
THEN 'Pediatric'

WHEN age BETWEEN 18 AND 64
THEN 'Adult'

ELSE 'Senior'

END AS age_group,

COUNT(*) AS total_patients,

ROUND(
AVG(total_claim_cost),2
) AS avg_claim_cost,

ROUND(
SUM(total_claim_cost),2
) AS total_population_cost

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY age_group

ORDER BY avg_claim_cost DESC;

age_group,total_patients,avg_claim_cost,total_population_cost
Senior,136,429171.49,5.836732282E7
Adult,339,217833.67,7.384561343E7
Pediatric,80,16535.91,1322873.17


# Query 3.2 Results

# Claim Cost by Age Group

---

## Findings

| Age Group | Avg Claim Cost | Total Population Cost |
|-----------|----------------:|-----------------------:|
| Senior | $429K | $58.4M |
| Adult | $218K | $73.8M |
| Pediatric | $16K | $1.3M |

---

# Major Findings

## Finding 1

Average healthcare spending increases with age.

Average claim cost:

Pediatric → $16K

Adult → $218K

Senior → $429K

Observed:

Senior populations cost substantially more per patient.

---

## Finding 2

Adult populations generate highest aggregate spending.

Total population spending:

Adult → $73.8M

Senior → $58.4M

Reason:

Larger population size

This suggests:

Large moderate-cost populations create substantial financial burden.

---

## Finding 3

Pediatric populations exhibit comparatively low financial burden.

Observed:

Lowest:

- average cost
- total cost

---

# Financial Interpretation

Healthcare spending is driven by:

High average cost:

Senior populations

High aggregate cost:

Adult populations

Both groups contribute differently to financial burden.

---

# Executive Summary

Older populations incur greater cost per patient, while adult populations generate the largest total spending through volume.

--------------------------------------------------

SECTION 3:
Financial Analytics

Completed:

✓ Query 3.1
Cost by Risk

✓ Query 3.2
Cost by Age

Status:

IN PROGRESS

Next:

Query 3.3

Cost by Chronic Disease Burden

--------------------------------------------------

# Query 3.3

# Claim Cost by Chronic Disease Burden

---

## Business Question

Do multiple chronic diseases increase healthcare spending?

---

## What We Are Doing

Group by:

chronic_disease_count

Then compare:

- average claim cost
- total population cost

---

## Why

Multiple chronic diseases often increase:

Medication burden

↓

Admissions

↓

Care complexity

↓

Healthcare cost

---

## Expected Output

Increasing chronic disease burden

↓

Increasing cost

---

## Business Value

Supports:

Cost containment

Population health programs

Care coordination

In [0]:
%sql
SELECT

chronic_disease_count,

COUNT(*) AS total_patients,

ROUND(
AVG(total_claim_cost),2
) AS avg_claim_cost,

ROUND(
SUM(total_claim_cost),2
) AS total_population_cost

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY chronic_disease_count

ORDER BY chronic_disease_count;

chronic_disease_count,total_patients,avg_claim_cost,total_population_cost
0,114,70504.41,8037502.26
1,90,204866.27,1.843796442E7
2,167,226873.81,3.788792585E7
3,115,205471.18,2.362918561E7
4,56,478598.2,2.680149909E7
5,11,441296.18,4854257.97
6,2,6943737.11,1.388747422E7


# Query 3.3 Results

# Claim Cost by Chronic Disease Burden

---

## Findings

| Chronic Diseases | Avg Claim Cost | Total Population Cost |
|-----------------|----------------:|-----------------------:|
| 0 | $70K | $8.0M |
| 1 | $205K | $18.4M |
| 2 | $227K | $37.9M |
| 3 | $205K | $23.6M |
| 4 | $479K | $26.8M |
| 5 | $441K | $4.9M |
| 6 | $6.94M | $13.9M |

---

# Major Findings

## Finding 1

Average healthcare spending generally rises with chronic disease burden.

Observed:

0 diseases → $70K

4 diseases → $479K

6 diseases → $6.94M

Patients with multiple chronic diseases exhibit dramatically greater financial burden.

---

## Finding 2

Extreme multimorbidity populations generate very high per-patient spending.

Observed:

6 chronic diseases

Average cost:

≈ $6.94M

This suggests:

Small complex populations drive extreme spending.

---

## Finding 3

Moderate disease burden populations generate large aggregate spending.

Example:

2 diseases

Total population spending:

≈ $37.9M

Reason:

Higher population size

---

# Financial Interpretation

Financial burden appears concentrated among:

- multimorbidity populations
- complex patients
- chronic disease populations

---

# Executive Summary

Increasing chronic disease burden is strongly associated with elevated healthcare spending and cost concentration.

--------------------------------------------------

SECTION 3:
Financial Analytics

Completed:

✓ Query 3.1
Cost by Risk

✓ Query 3.2
Cost by Age

✓ Query 3.3
Cost by Chronic Disease Burden

Status:

IN PROGRESS

Next:

Query 3.4

Identify Highest Cost Patients

--------------------------------------------------

# Query 3.4

# Highest Cost Patient Analysis

---

## Business Question

Which patients generate the greatest healthcare spending?

---

## What We Are Doing

Rank patients by:

total_claim_cost

Then evaluate:

- risk category
- age
- chronic disease burden
- utilization

---

## Why

Healthcare spending often follows:

Small population

↓

Extreme cost concentration

We want to identify:

High-cost populations

---

## Expected Output

Top cost-generating patients

---

## Business Value

Supports:

Cost reduction

Care management

Financial forecasting

In [0]:
%sql
SELECT

patient_id,

population_health_risk_category,

age,

chronic_disease_count,

total_encounters,

total_claim_cost,

emergency_encounters,

inpatient_encounters

FROM healthcare_catalog.gold.population_health_dashboard

ORDER BY total_claim_cost DESC

LIMIT 10;

patient_id,population_health_risk_category,age,chronic_disease_count,total_encounters,total_claim_cost,emergency_encounters,inpatient_encounters
e23c7a6e-6ef2-3932-1361-b73b4c8a3961,High Risk,111,6,705,1.3535234989999948E7,29,0
55c5b8d3-99d0-58ad-8444-e42bb81bd5c7,High Risk,84,4,412,1.0331274649999976E7,23,0
ef167059-cef0-12c4-49db-993ca3a20c01,High Risk,97,4,1563,2190959.000000004,5,17
c03807d7-0b42-0490-23f0-8caeb7c38e74,High Risk,99,4,111,1597934.100000001,4,67
1f528581-7613-2b96-dce3-68c7cf967ea1,Medium Risk,41,2,112,1427488.81,2,0
454bbec4-42bc-bd2c-fe3a-f8f4fc4346e9,High Risk,24,2,120,1241790.09,3,0
10a1709a-5dde-1e3a-4c06-d8de03764712,Medium Risk,68,3,80,1211655.7099999997,2,48
7ec518e6-787e-cc8a-28f2-6da51fe584b8,Medium Risk,99,2,55,1114564.2399999993,1,0
da97668c-bf70-6c4d-3157-24aae70ed237,Medium Risk,97,1,138,1084982.6500000006,2,23
7bc66758-5d59-0b38-1532-29a33fa146c1,High Risk,114,3,158,1064065.9899999993,6,5


# Query 3.4 Results

# Highest Cost Patient Analysis

---

## Findings

Highest cost populations exhibit:

- elevated risk category
- advanced age
- multimorbidity
- high utilization
- admission burden

---

# Major Findings

## Finding 1

Extreme healthcare spending is concentrated among high-risk populations.

Most top-cost patients belong to:

High Risk category

Observed:

High-risk populations dominate financial burden rankings.

---

## Finding 2

Multimorbidity contributes substantially to cost burden.

Top cost patients commonly exhibit:

4 chronic diseases

6 chronic diseases

Complex disease burden appears associated with elevated spending.

---

## Finding 3

High utilization often coincides with extreme spending.

Examples:

705 encounters

↓

$13.5M claims

412 encounters

↓

$10.3M claims

Observed relationship:

Higher utilization

↓

Higher spending

---

## Finding 4

Advanced age populations contribute to elevated costs.

Many top-cost patients:

Age >80

This aligns with previous findings:

Older populations generate greater healthcare burden.

---

# Financial Interpretation

High-cost populations may require:

✓ care coordination

✓ chronic disease management

✓ cost reduction initiatives

✓ targeted intervention programs

---

# Executive Summary

Healthcare spending is highly concentrated among older, high-risk, multimorbidity populations exhibiting elevated utilization and complex care needs.

--------------------------------------------------

SECTION COMPLETE

SECTION:

Financial Analytics

Completed Analyses:

✓ Cost by Risk Category

✓ Cost by Age Group

✓ Cost by Chronic Disease Burden

✓ Highest Cost Patients

---

# Main Findings

1.

High-risk populations generate highest average spending.

2.

Senior populations exhibit highest cost per patient.

3.

Multimorbidity substantially increases financial burden.

4.

Small populations drive extreme spending concentration.

---

# Financial Impact

Supports:

- cost containment
- payer analytics
- financial planning
- population health management

---

# Outputs Generated

Financial burden KPIs

Cost concentration metrics

High-cost population identification

Cost driver insights

---

Status:

SECTION 3 COMPLETE

Proceeding to:

SECTION 4

Cost Driver Analytics

--------------------------------------------------

# SECTION 4

# Cost Driver Analytics

Status:

STARTED

---

# Objective

Identify variables contributing most to healthcare spending.

Potential cost drivers:

- utilization burden
- chronic disease burden
- risk category
- age
- admissions
- emergency utilization
- medication burden

This section answers:

> Which factors increase healthcare cost?

---

# Why This Matters

Healthcare organizations use cost-driver analysis for:

✓ Cost reduction

✓ Care management

✓ Risk stratification

✓ Population health programs

✓ Value-based reimbursement

---

# Expected Outputs

Cost driver KPIs:

- cost vs utilization
- cost vs multimorbidity
- cost vs admissions
- cost vs medication burden

Executive insights:

Drivers of spending concentration

# Query 4.1

# Utilization Burden vs Healthcare Spending

---

## Business Question

Do patients with more encounters generate higher spending?

---

## What We Are Doing

Group patients by:

total_encounters

Then compare:

Average claim cost

---

## Why

Healthcare utilization often drives:

Admissions

↓

Procedures

↓

Treatment

↓

Claims cost

We want to determine:

Does higher utilization increase spending?

---

## Expected Output

Increasing encounters

↓

Increasing cost

---

## Business Value

Supports:

Utilization reduction programs

Cost containment initiatives

In [0]:
%sql
SELECT

CASE

WHEN total_encounters < 25
THEN 'Low Utilization'

WHEN total_encounters BETWEEN 25 AND 75
THEN 'Moderate Utilization'

WHEN total_encounters BETWEEN 76 AND 150
THEN 'High Utilization'

ELSE 'Extreme Utilization'

END AS utilization_group,

COUNT(*) AS total_patients,

ROUND(
AVG(total_claim_cost),2
) AS avg_claim_cost,

ROUND(
AVG(total_encounters),2
) AS avg_encounters

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY utilization_group

ORDER BY avg_claim_cost DESC;

utilization_group,total_patients,avg_claim_cost,avg_encounters
Extreme Utilization,7,4011645.78,546.0
High Utilization,76,510064.68,98.79
Moderate Utilization,288,218177.46,46.64
Low Utilization,184,20947.1,16.58


# Query 4.1 Results

# Utilization Burden vs Healthcare Spending

---

## Findings

| Utilization Group | Avg Encounters | Avg Claim Cost |
|------------------|---------------:|----------------:|
| Extreme Utilization | 546.00 | $4.01M |
| High Utilization | 98.79 | $510K |
| Moderate Utilization | 46.64 | $218K |
| Low Utilization | 16.58 | $21K |

---

# Major Findings

## Finding 1

Healthcare spending increases sharply with utilization burden.

Observed:

Low Utilization:

≈ $21K average cost

Extreme Utilization:

≈ $4.0M average cost

This suggests:

Utilization is a major cost driver.

---

## Finding 2

Extreme utilization populations generate disproportionate financial burden.

Observed:

Only:

7 patients

Yet:

Average spending:

≈ $4M

Small populations create extreme costs.

---

## Finding 3

Encounter burden appears strongly associated with spending escalation.

Observed relationship:

Increasing encounters

↓

Increasing healthcare cost

---

# Cost Driver Interpretation

Utilization burden likely contributes to:

- admissions
- procedures
- treatments
- claim costs

---

# Executive Summary

Healthcare utilization is one of the strongest predictors of financial burden within the population.

--------------------------------------------------

SECTION 4:
Cost Driver Analytics

Completed:

✓ Query 4.1
Utilization vs Cost

Status:

IN PROGRESS

Next:

Query 4.2

Admission Burden vs Cost

--------------------------------------------------

# Query 4.2

# Inpatient Burden vs Healthcare Spending

---

## Business Question

Do patients with greater inpatient burden generate higher spending?

---

## What We Are Doing

Group by:

inpatient_encounters

Then compare:

Average claim cost

---

## Why

Admissions often increase:

Treatment complexity

↓

Length of stay

↓

Procedures

↓

Healthcare spending

---

## Expected Output

More admissions

↓

Higher cost

---

## Business Value

Supports:

Hospital utilization programs

Cost containment strategies

In [0]:
%sql
SELECT

CASE

WHEN inpatient_encounters = 0
THEN 'No Admission'

WHEN inpatient_encounters BETWEEN 1 AND 5
THEN 'Low Admission'

WHEN inpatient_encounters BETWEEN 6 AND 20
THEN 'Moderate Admission'

ELSE 'High Admission'

END AS admission_group,

COUNT(*) AS total_patients,

ROUND(
AVG(total_claim_cost),2
) AS avg_claim_cost,

ROUND(
AVG(inpatient_encounters),2
) AS avg_inpatient

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY admission_group

ORDER BY avg_claim_cost DESC;

admission_group,total_patients,avg_claim_cost,avg_inpatient
High Admission,11,680765.54,41.45
Moderate Admission,12,414652.01,9.25
Low Admission,89,268518.17,1.46
No Admission,443,219353.15,0.0


# Query 4.2 Results

# Inpatient Burden vs Healthcare Spending

---

## Findings

| Admission Group | Avg Inpatient | Avg Claim Cost |
|----------------|---------------:|----------------:|
| High Admission | 41.45 | $681K |
| Moderate Admission | 9.25 | $415K |
| Low Admission | 1.46 | $269K |
| No Admission | 0.00 | $219K |

---

# Major Findings

## Finding 1

Average healthcare spending increases with admission burden.

Observed:

No Admission:

≈ $219K

High Admission:

≈ $681K

Patients with higher admission burden generate elevated spending.

---

## Finding 2

Admission burden contributes to financial escalation.

Observed relationship:

Higher admissions

↓

Higher treatment burden

↓

Higher healthcare cost

---

## Finding 3

Admission burden appears weaker than total utilization burden.

Previous analysis:

Extreme utilization

≈ $4M average cost

Current analysis:

High admission burden

≈ $681K average cost

This suggests:

Overall utilization may be a stronger cost driver than admissions alone.

---

# Cost Driver Interpretation

Admissions likely increase:

- inpatient treatment
- procedures
- monitoring
- resource consumption

However:

Total utilization may better explain spending variation.

---

# Executive Summary

Admission burden contributes to healthcare spending, although overall utilization appears to be a stronger predictor of cost escalation.

--------------------------------------------------

SECTION 4:
Cost Driver Analytics

Completed:

✓ Query 4.1
Utilization vs Cost

✓ Query 4.2
Admission vs Cost

Status:

IN PROGRESS

Next:

Query 4.3

Multimorbidity vs Cost

--------------------------------------------------

# Query 4.3

# Chronic Disease Burden vs Healthcare Spending

---

## Business Question

Does multimorbidity increase healthcare cost?

---

## What We Are Doing

Group by:

chronic_disease_count

Compare:

Average claim cost

---

## Why

Multiple chronic diseases increase:

Medication burden

↓

Care complexity

↓

Utilization

↓

Cost

---

## Expected Output

More chronic diseases

↓

Higher spending

---

## Business Value

Supports:

Population health programs

Care management

Cost reduction initiatives

In [0]:
%sql
SELECT

CASE

WHEN chronic_disease_count = 0
THEN 'No Chronic Disease'

WHEN chronic_disease_count BETWEEN 1 AND 2
THEN 'Low Burden'

WHEN chronic_disease_count BETWEEN 3 AND 4
THEN 'Moderate Burden'

ELSE 'High Burden'

END AS disease_group,

COUNT(*) AS total_patients,

ROUND(
AVG(total_claim_cost),2
) AS avg_claim_cost,

ROUND(
AVG(chronic_disease_count),2
) AS avg_disease_count

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY disease_group

ORDER BY avg_claim_cost DESC;

disease_group,total_patients,avg_claim_cost,avg_disease_count
High Burden,13,1441671.71,5.15
Moderate Burden,171,294916.28,3.33
Low Burden,257,219166.89,1.65
No Chronic Disease,114,70504.41,0.0


# Query 4.3 Results

# Chronic Disease Burden vs Healthcare Spending

---

## Findings

| Disease Group | Avg Disease Count | Avg Claim Cost |
|---------------|------------------:|----------------:|
| High Burden | 5.15 | $1.44M |
| Moderate Burden | 3.33 | $295K |
| Low Burden | 1.65 | $219K |
| No Chronic Disease | 0.00 | $70K |

---

# Major Findings

## Finding 1

Healthcare spending rises substantially with chronic disease burden.

Observed:

No Chronic Disease:

≈ $70K average cost

High Burden:

≈ $1.44M average cost

Approximate increase:

≈ 20x greater spending

---

## Finding 2

Multimorbidity appears to be a major financial driver.

Observed relationship:

More chronic disease

↓

Greater complexity

↓

More utilization

↓

Higher cost

---

## Finding 3

Patients with high disease burden generate extreme financial burden despite small population size.

Observed:

High Burden population:

13 patients

Average spending:

≈ $1.44M

Small populations drive disproportionate spending.

---

# Cost Driver Interpretation

Multimorbidity contributes to:

- utilization escalation
- treatment complexity
- medication burden
- healthcare cost concentration

---

# Executive Summary

Chronic disease burden is strongly associated with increased healthcare spending and represents a major cost driver within the population.

--------------------------------------------------

SECTION 4:
Cost Driver Analytics

Completed:

✓ Query 4.1
Utilization vs Cost

✓ Query 4.2
Admission vs Cost

✓ Query 4.3
Multimorbidity vs Cost

Status:

IN PROGRESS

Next:

Query 4.4

Medication Burden vs Cost

--------------------------------------------------

# Query 4.4

# Medication Burden vs Healthcare Spending

---

## Business Question

Do patients with higher medication burden generate greater spending?

---

## What We Are Doing

Group patients by:

polypharmacy_flag

Then compare:

Average claim cost

---

## Why

Medication burden may indicate:

Complex disease

↓

Greater treatment need

↓

Higher spending

---

## Expected Output

Polypharmacy populations

↓

Higher cost

---

## Business Value

Supports:

Medication management

Population health programs

Cost reduction initiatives

In [0]:
%sql
SELECT

CASE

WHEN polypharmacy_flag = 1
THEN 'Polypharmacy'

ELSE 'Non-Polypharmacy'

END AS medication_group,

COUNT(*) AS total_patients,

ROUND(
AVG(total_claim_cost),2
) AS avg_claim_cost,

ROUND(
AVG(total_medication_requests),2
) AS avg_medications

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY medication_group

ORDER BY avg_claim_cost DESC;

medication_group,total_patients,avg_claim_cost,avg_medications
Polypharmacy,260,390461.29,79.13
Non-Polypharmacy,295,108528.39,12.48


# Query 4.4 Results

# Medication Burden vs Healthcare Spending

---

## Findings

| Medication Group | Avg Medications | Avg Claim Cost |
|-----------------|----------------:|----------------:|
| Polypharmacy | 79.13 | $390K |
| Non-Polypharmacy | 12.48 | $109K |

---

# Major Findings

## Finding 1

Healthcare spending increases substantially among polypharmacy populations.

Observed:

Non-Polypharmacy:

≈ $109K average cost

Polypharmacy:

≈ $390K average cost

Approximate increase:

≈ 3.6x greater spending

---

## Finding 2

Medication burden appears associated with elevated financial burden.

Observed relationship:

More medications

↓

Greater complexity

↓

Higher healthcare spending

---

## Finding 3

Polypharmacy may indicate:

- multimorbidity
- chronic disease burden
- complex treatment pathways

These populations exhibit elevated cost burden.

---

# Cost Driver Interpretation

Medication burden likely contributes to:

- treatment complexity
- utilization burden
- healthcare spending

---

# Executive Summary

Polypharmacy populations exhibit substantially higher healthcare spending and represent a potential target for medication optimization strategies.

--------------------------------------------------

SECTION COMPLETE

SECTION:

Cost Driver Analytics

Completed Analyses:

✓ Utilization vs Cost

✓ Admission Burden vs Cost

✓ Multimorbidity vs Cost

✓ Medication Burden vs Cost

---

# Main Cost Drivers Identified

Strongest observed drivers:

1.

Extreme utilization burden

Average cost:

≈ $4.0M

---

2.

Multimorbidity

Average cost:

≈ $1.44M

---

3.

Polypharmacy

Average cost:

≈ $390K

---

4.

Admission burden

Average cost:

≈ $681K

---

# Cost Driver Ranking (Observed)

Strongest → Weakest

Utilization burden

↓

Multimorbidity

↓

Admission burden

↓

Medication burden

---

# Business Impact

Supports:

- cost containment
- utilization reduction
- medication optimization
- care coordination
- population health management

---

# Outputs Generated

Cost driver KPIs

Financial burden insights

Population segmentation

High-cost population identification

---

Status:

SECTION 4 COMPLETE

Proceeding to:

SECTION 5

Comparative + Ranking Analytics

--------------------------------------------------

# SECTION 5

# Comparative + Ranking Analytics

Status:

STARTED

---

# Objective

Compare patient populations and rank high-impact groups.

Focus:

- top spenders
- top utilizers
- highest burden populations
- comparative risk analysis

---

# Why This Matters

Healthcare organizations often ask:

Who should be prioritized?

Which populations drive burden?

Where should intervention begin?

---

# Expected Outputs

Ranking KPIs

Comparative metrics

Executive summaries

Dashboard-ready outputs

# Query 5.1

# Comparative Spending by Risk Category

---

## Business Question

Which risk populations generate greatest total financial burden?

---

## What We Are Doing

Compare:

population_health_risk_category

against:

Total spending

Average spending

---

## Why

Average cost and total cost may show different patterns.

Example:

Small populations → High average cost

Large populations → High aggregate cost

---

## Expected Output

Risk groups ranked by:

Total population spending

In [0]:
%sql
SELECT

population_health_risk_category,

COUNT(*) AS total_patients,

ROUND(
AVG(total_claim_cost),2
) AS avg_claim_cost,

ROUND(
SUM(total_claim_cost),2
) AS total_population_cost,

RANK() OVER(

ORDER BY
SUM(total_claim_cost) DESC

) AS spending_rank

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY population_health_risk_category

ORDER BY spending_rank;

population_health_risk_category,total_patients,avg_claim_cost,total_population_cost,spending_rank
Medium Risk,231,270689.56,6.252928741E7,1
High Risk,77,748090.65,5.760297976E7,2
Low Risk,247,54265.35,1.340354225E7,3


# Query 5.2

# Top 10 Highest Cost Patients

---

## Business Question

Which patients generate greatest healthcare spending?

---

## What We Are Doing

Rank patients by:

total_claim_cost

---

## Why

Healthcare spending is often concentrated.

A small population drives large costs.

---

## Expected Output

Top cost populations with ranking

---

## Business Value

Supports:

Care management

Cost reduction

Population prioritization

In [0]:
%sql
SELECT

patient_id,

population_health_risk_category,

age,

chronic_disease_count,

total_claim_cost,

RANK() OVER(

ORDER BY
total_claim_cost DESC

) AS cost_rank

FROM healthcare_catalog.gold.population_health_dashboard

ORDER BY cost_rank

LIMIT 10;

patient_id,population_health_risk_category,age,chronic_disease_count,total_claim_cost,cost_rank
e23c7a6e-6ef2-3932-1361-b73b4c8a3961,High Risk,111,6,1.3535234989999948E7,1
55c5b8d3-99d0-58ad-8444-e42bb81bd5c7,High Risk,84,4,1.0331274649999976E7,2
ef167059-cef0-12c4-49db-993ca3a20c01,High Risk,97,4,2190959.000000004,3
c03807d7-0b42-0490-23f0-8caeb7c38e74,High Risk,99,4,1597934.100000001,4
1f528581-7613-2b96-dce3-68c7cf967ea1,Medium Risk,41,2,1427488.81,5
454bbec4-42bc-bd2c-fe3a-f8f4fc4346e9,High Risk,24,2,1241790.09,6
10a1709a-5dde-1e3a-4c06-d8de03764712,Medium Risk,68,3,1211655.7099999997,7
7ec518e6-787e-cc8a-28f2-6da51fe584b8,Medium Risk,99,2,1114564.2399999993,8
da97668c-bf70-6c4d-3157-24aae70ed237,Medium Risk,97,1,1084982.6500000006,9
7bc66758-5d59-0b38-1532-29a33fa146c1,High Risk,114,3,1064065.9899999993,10


# Query 5.2 Results

# Top 10 Highest Cost Patients

---

## Findings

Top cost populations frequently exhibit:

- High Risk status
- Advanced age
- Multiple chronic diseases
- Elevated utilization burden

---

# Major Findings

## Finding 1

High-risk populations dominate top cost rankings.

Observed:

Most Top-10 patients belong to:

High Risk category

---

## Finding 2

Advanced age appears strongly associated with elevated spending.

Examples:

Age:

111

99

97

114

Older populations frequently appear among highest-cost patients.

---

## Finding 3

Multimorbidity contributes to cost concentration.

Observed:

High-cost populations often exhibit:

4–6 chronic diseases

---

# Comparative Interpretation

Cost concentration appears associated with:

Higher risk

+

Older age

+

Multimorbidity

↓

Extreme financial burden

---

# Executive Summary

Healthcare spending is concentrated among older, high-risk, multimorbidity populations exhibiting elevated clinical complexity.

--------------------------------------------------

SECTION 5:
Comparative + Ranking Analytics

Completed:

✓ Comparative Spending by Risk

✓ Top Cost Patient Ranking

Status:

IN PROGRESS

Next:

Query 5.3

Top Utilization Patient Ranking

--------------------------------------------------

# Query 5.3

# Top Utilization Patient Ranking

---

## Business Question

Which patients consume greatest healthcare resources?

---

## What We Are Doing

Rank by:

total_encounters

---

## Why

Utilization concentration often drives:

Operational burden

↓

Financial burden

---

## Expected Output

Top utilization populations ranked

---

## Business Value

Supports:

Population prioritization

Care management

Utilization reduction

In [0]:
%sql
SELECT

patient_id,

population_health_risk_category,

age,

chronic_disease_count,

total_encounters,

RANK() OVER(

ORDER BY
total_encounters DESC

) AS utilization_rank

FROM healthcare_catalog.gold.population_health_dashboard

ORDER BY utilization_rank

LIMIT 10;

patient_id,population_health_risk_category,age,chronic_disease_count,total_encounters,utilization_rank
ef167059-cef0-12c4-49db-993ca3a20c01,High Risk,97,4,1563,1
e23c7a6e-6ef2-3932-1361-b73b4c8a3961,High Risk,111,6,705,2
8c6ae452-5f8c-9ff6-006d-c6c860acf5cd,High Risk,108,5,638,3
55c5b8d3-99d0-58ad-8444-e42bb81bd5c7,High Risk,84,4,412,4
eb7cffc3-28d2-e8b0-72bc-8e86860ed313,High Risk,64,4,182,5
3dfce025-b696-711a-8c2d-6903736e2882,High Risk,63,4,164,6
7bc66758-5d59-0b38-1532-29a33fa146c1,High Risk,114,3,158,7
8bf6aa92-645f-3c82-ddcd-5851496a6aa8,High Risk,59,4,145,8
a8c08d9f-a2ae-46c6-0371-f596ed13d1cf,Medium Risk,99,5,140,9
72f91cc7-fb63-84e8-32ff-86456bfa446e,High Risk,83,3,139,10


# Query 5.3 Results

# Top Utilization Patient Ranking

---

## Findings

Highest utilization populations frequently exhibit:

- High Risk status
- Advanced age
- Elevated chronic disease burden

---

# Major Findings

## Finding 1

High-risk populations dominate utilization rankings.

Observed:

Most Top-10 utilizers belong to:

High Risk category

---

## Finding 2

Advanced age appears strongly associated with extreme utilization.

Examples:

Age:

97

108

111

114

Older populations consume substantially greater healthcare resources.

---

## Finding 3

Multimorbidity contributes to utilization concentration.

Observed:

Top utilizers frequently exhibit:

3–6 chronic diseases

---

## Finding 4

Utilization concentration appears associated with cost concentration.

Several patients identified previously as:

Highest cost

also appear among:

Highest utilization

Observed relationship:

Higher utilization

↓

Higher spending

---

# Comparative Interpretation

Resource burden appears concentrated among:

High risk

+

Older populations

+

Multimorbidity

↓

Extreme utilization

---

# Executive Summary

Healthcare utilization is highly concentrated among older, high-risk, multimorbidity populations, contributing to operational and financial burden.

--------------------------------------------------

SECTION 5:
Comparative + Ranking Analytics

Completed:

✓ Comparative Spending by Risk

✓ Top Cost Patient Ranking

✓ Top Utilization Patient Ranking

Status:

IN PROGRESS

Next:

Query 5.4

Compare Cost Rank vs Utilization Rank

--------------------------------------------------

# Query 5.4

# Cost Rank vs Utilization Rank Comparison

---

## Business Question

Do highest utilizers also generate highest spending?

---

## What We Are Doing

Compare:

Cost ranking

vs

Utilization ranking

---

## Why

If utilization strongly predicts cost:

High utilizers

↓

High spenders

This helps identify intervention targets.

---

## Expected Output

Top populations ranked by:

cost

utilization

---

## Business Value

Supports:

Population prioritization

Cost reduction

Care management

In [0]:
%sql
WITH ranked_patients AS (

SELECT

patient_id,

population_health_risk_category,

age,

total_encounters,

total_claim_cost,

RANK() OVER(
ORDER BY total_claim_cost DESC
) AS cost_rank,

RANK() OVER(
ORDER BY total_encounters DESC
) AS utilization_rank

FROM healthcare_catalog.gold.population_health_dashboard

)

SELECT *

FROM ranked_patients

WHERE cost_rank <=10
OR utilization_rank <=10

ORDER BY cost_rank;

patient_id,population_health_risk_category,age,total_encounters,total_claim_cost,cost_rank,utilization_rank
e23c7a6e-6ef2-3932-1361-b73b4c8a3961,High Risk,111,705,1.3535234989999948E7,1,2
55c5b8d3-99d0-58ad-8444-e42bb81bd5c7,High Risk,84,412,1.0331274649999976E7,2,4
ef167059-cef0-12c4-49db-993ca3a20c01,High Risk,97,1563,2190959.000000004,3,1
c03807d7-0b42-0490-23f0-8caeb7c38e74,High Risk,99,111,1597934.100000001,4,27
1f528581-7613-2b96-dce3-68c7cf967ea1,Medium Risk,41,112,1427488.81,5,26
454bbec4-42bc-bd2c-fe3a-f8f4fc4346e9,High Risk,24,120,1241790.09,6,23
10a1709a-5dde-1e3a-4c06-d8de03764712,Medium Risk,68,80,1211655.7099999997,7,73
7ec518e6-787e-cc8a-28f2-6da51fe584b8,Medium Risk,99,55,1114564.2399999993,8,166
da97668c-bf70-6c4d-3157-24aae70ed237,Medium Risk,97,138,1084982.6500000006,9,12
7bc66758-5d59-0b38-1532-29a33fa146c1,High Risk,114,158,1064065.9899999993,10,7


# Query 5.4 Results

# Cost Rank vs Utilization Rank Comparison

---

## Findings

Observed overlap:

Several highest-cost patients also appear among highest-utilization patients.

Examples:

| Cost Rank | Utilization Rank |
|-----------|------------------|
| 1 | 2 |
| 2 | 4 |
| 3 | 1 |
| 10 | 7 |

---

# Major Findings

## Finding 1

Utilization burden contributes substantially to spending.

Observed:

Top cost populations frequently exhibit:

High utilization ranking

---

## Finding 2

Utilization alone does not fully explain cost.

Examples:

Patient:

638 encounters

Cost Rank:

111

This suggests:

Additional factors influence spending.

Potential contributors:

- treatment complexity
- procedures
- chronic disease burden
- medications

---

## Finding 3

Cost burden appears multifactorial.

Observed drivers:

✓ utilization

✓ multimorbidity

✓ risk category

✓ medication burden

---

# Comparative Interpretation

Healthcare spending is partially driven by utilization but also influenced by clinical complexity.

---

# Executive Summary

Utilization is a major predictor of cost burden; however, additional clinical factors contribute substantially to spending concentration.

--------------------------------------------------

SECTION COMPLETE

SECTION:

Comparative + Ranking Analytics

Completed Analyses:

✓ Comparative Spending by Risk

✓ Highest Cost Patient Ranking

✓ Highest Utilization Ranking

✓ Cost Rank vs Utilization Rank Comparison

---

# Main Findings

1.

Medium-risk populations generate highest aggregate spending.

2.

High-risk populations dominate cost rankings.

3.

High-risk populations dominate utilization rankings.

4.

Utilization predicts spending but does not fully explain cost.

---

# Outputs Generated

Ranking metrics

Comparative analytics

Population prioritization insights

Executive dashboard KPIs

---

Status:

SECTION 5 COMPLETE

--------------------------------------------------

# NOTEBOOK COMPLETE

Notebook:

22_sql_healthcare_operations_analytics

Status:

Completed

---

# Sections Completed

✓ Resource Utilization Analytics

✓ Healthcare Operations Analytics

✓ Financial Analytics

✓ Cost Driver Analytics

✓ Comparative + Ranking Analytics

---

# Major Findings

1.

High-risk populations consume ~5x greater healthcare resources.

---

2.

Senior populations generate elevated operational burden.

---

3.

Multimorbidity substantially increases:

- utilization
- admissions
- cost

---

4.

Extreme utilization is the strongest observed cost driver.

---

5.

Healthcare spending is concentrated among:

Older

High-risk

Multimorbidity populations

---

# Technologies Used

SQL

Spark SQL

Window Functions

CASE WHEN

GROUP BY

RANK()

CTE

Aggregation Functions

---

# Output Artifacts

Generated:

Operational KPIs

Financial KPIs

Cost Driver Metrics

Ranking Analytics

Dashboard-ready Measures

---

# Downstream Consumers

Notebook 23

Dashboard development

Predictive modeling

Clinical AI

---

# Executive Summary

Healthcare burden is disproportionately generated by small high-risk populations exhibiting elevated utilization, multimorbidity, and financial burden.

Export sample rows from Databricks

In [0]:
%sql
SELECT *

FROM healthcare_catalog.gold.population_health_dashboard

LIMIT 100

patient_id,gender,age,city,state,marital_status,total_encounters,ambulatory_encounters,emergency_encounters,inpatient_encounters,avg_encounter_duration_hours,total_encounter_hours,high_emergency_utilization_flag,high_inpatient_utilization_flag,high_total_utilization_flag,diabetes_flag,hypertension_flag,asthma_flag,coronary_heart_disease_flag,chronic_kidney_disease_flag,obesity_flag,mental_health_flag,copd_flag,stroke_flag,cancer_flag,chronic_disease_count,multi_chronic_condition_flag,total_medication_requests,unique_medications,polypharmacy_flag,high_medication_burden_flag,total_observations,avg_bmi,avg_glucose,avg_hba1c,avg_heart_rate,avg_respiratory_rate,avg_oxygen_saturation,obesity_from_bmi_flag,high_glucose_flag,high_hba1c_flag,low_oxygen_saturation_flag,total_procedures,unique_procedure_count,total_careplans,active_careplans,completed_careplans,high_procedure_burden_flag,active_care_management_flag,high_treatment_complexity_flag,total_claims,total_claim_cost,avg_claim_cost,max_claim_cost,pharmacy_claim_cost,institutional_claim_cost,high_cost_patient_flag,very_high_cost_patient_flag,high_pharmacy_cost_flag,high_institutional_cost_flag,population_health_risk_score,population_health_risk_category
b0a06ead-cc42-aa48-dad6-841d4aa679fa,male,73,Boston,MA,M,32,30,2,0,0.30257812500000003,9.682500000000001,0,0,0,1,0,0,1,0,0,1,0,0,0,3,1,53,7,1,1,289,29.53833333333333,81.855,6.031666666666666,83.41666666666667,13.916666666666666,0.0,0,0,0,0,73,15,7,3,4,1,1,1,85,120811.62000000001,1421.3131764705884,89974.97,4057.2399999999993,116754.38,1,0,0,1,4,Medium Risk
ccfc4db2-2026-7adb-3db0-33f3828140bb,male,49,Peabody,MA,M,38,37,1,0,0.26973684210526316,10.25,0,0,0,0,1,0,0,0,1,1,0,0,0,3,1,49,4,0,1,138,28.428888888888892,0.0,0.0,78.55555555555556,14.333333333333334,0.0,0,0,0,0,44,9,4,2,2,1,1,1,87,40134.75000000001,461.31896551724145,1967.13,10271.510000000002,29863.240000000016,0,0,1,0,2,Low Risk
31a2e8ec-69fc-8a71-3ab6-36cbdd508713,female,108,Quincy,MA,M,64,63,0,1,0.62109375,39.75,0,0,1,0,0,0,0,0,1,1,0,0,0,2,1,1,1,0,0,175,27.430000000000003,0.0,0.0,84.45454545454545,14.454545454545455,0.0,0,0,0,0,65,11,1,1,0,1,1,1,65,208955.26999999973,3214.6964615384572,64760.45,0.0,208955.26999999973,1,0,0,1,3,Medium Risk
71a8b156-760b-df6b-859e-eefc7932a526,female,11,Boston,MA,Never Married,19,18,1,0,0.2894736842105263,5.5,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,4,3,0,0,175,58.95464285714286,0.0,0.0,81.125,13.5,0.0,1,0,0,0,9,2,1,0,1,0,0,0,23,22147.81000000001,962.9482608695656,10551.46,130.12,22017.690000000006,0,0,0,0,0,Low Risk
76b289fd-e825-734c-8446-316f59643593,female,41,Boston,MA,M,24,21,2,1,9.409189814814814,225.82055555555556,0,0,0,1,0,0,0,0,0,1,0,0,0,2,1,12,7,1,1,564,28.06,112.91999999999997,7.033333333333332,119.62471428571429,21.39764285714286,82.39636363636363,0,0,1,1,43,19,7,2,5,1,1,1,36,126532.93000000005,3514.8036111111123,60259.75000000001,104.03,126428.90000000005,1,0,0,1,5,High Risk
92fb7efc-5cfd-f8d3-927b-42f8ee099531,male,12,Worcester,MA,Never Married,27,25,2,0,0.35032921810699585,9.458888888888888,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,7,5,1,0,216,60.19681249999999,0.0,0.0,84.21222222222222,15.23361111111111,86.97,1,0,0,1,13,6,5,1,4,0,1,1,34,15155.11,445.7385294117647,1448.57,70.5,15084.610000000002,0,0,0,0,3,Medium Risk
81aa7647-779f-fd6b-94cf-782e606efeb2,female,11,Weston,MA,Never Married,17,17,0,0,0.25,4.25,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,0,0,174,25.2065,0.0,0.0,78.3125,13.875,0.0,0,0,0,0,7,2,1,0,1,0,0,0,18,10938.01,607.6672222222222,1177.28,5.96,10932.05,0,0,0,0,0,Low Risk
346a1435-2455-914f-c287-7b88052d05db,female,44,Framingham,MA,M,69,65,4,0,0.3047342995169082,21.026666666666667,1,0,1,1,1,0,0,0,0,1,0,0,0,3,1,57,8,1,1,241,29.746000000000002,81.597,6.089,75.4,14.5,0.0,0,0,0,0,179,43,7,3,4,1,1,1,126,605520.1200000002,4805.71523809524,69511.58999999998,140.59999999999994,605379.5200000001,1,1,0,1,5,High Risk
1cfa5a70-7f3c-4227-5cf1-e182fcff4cd4,female,67,Quincy,MA,M,38,37,1,0,5.659824561403509,215.07333333333335,0,0,0,1,0,0,0,0,0,1,0,0,0,2,1,26,4,0,

#1. Utilization Dashboard Data

In [0]:
%sql
SELECT

population_health_risk_category,

COUNT(*) total_patients,

AVG(total_encounters) avg_total_encounters,

AVG(emergency_encounters) avg_emergency,

AVG(inpatient_encounters) avg_inpatient

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY population_health_risk_category

population_health_risk_category,total_patients,avg_total_encounters,avg_emergency,avg_inpatient
Medium Risk,231,53.134199134199136,1.7012987012987013,0.8354978354978355
Low Risk,247,24.874493927125506,0.7327935222672065,0.05263157894736842
High Risk,77,122.0,5.857142857142857,6.376623376623376


#2. Operations Dashboard Data

In [0]:
%sql
SELECT

CASE

WHEN age<18 THEN 'Pediatric'
WHEN age BETWEEN 18 AND 64 THEN 'Adult'
ELSE 'Senior'

END age_group,

AVG(inpatient_encounters) avg_inpatient,

AVG(avg_encounter_duration_hours) avg_duration

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY age_group

age_group,avg_inpatient,avg_duration
Senior,2.9411764705882355,2.9005767541560274
Adult,0.8731563421828908,1.9203573217022056
Pediatric,0.0125,0.33399559656946815


# 3. Financial Dashboard Data

In [0]:
%sql
SELECT

population_health_risk_category,

AVG(total_claim_cost) avg_claim_cost,

SUM(total_claim_cost) total_claim_cost

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY population_health_risk_category

population_health_risk_category,avg_claim_cost,total_claim_cost
Medium Risk,270689.555887446,6.2529287410000026E7
Low Risk,54265.353238866395,1.340354225E7
High Risk,748090.6462337653,5.760297975999992E7


# 4. Cost Driver Dashboard Data

In [0]:
%sql
SELECT

polypharmacy_flag,

AVG(total_claim_cost)

FROM healthcare_catalog.gold.population_health_dashboard

GROUP BY polypharmacy_flag

polypharmacy_flag,AVG(total_claim_cost)
1,390461.2869999995
0,108528.38915254237


#5. Ranking Dashboard Data

In [0]:
%sql
SELECT

patient_id,

total_claim_cost,

total_encounters,

population_health_risk_category

FROM healthcare_catalog.gold.population_health_dashboard

ORDER BY total_claim_cost DESC

LIMIT 20

patient_id,total_claim_cost,total_encounters,population_health_risk_category
e23c7a6e-6ef2-3932-1361-b73b4c8a3961,1.3535234989999948E7,705,High Risk
55c5b8d3-99d0-58ad-8444-e42bb81bd5c7,1.0331274649999976E7,412,High Risk
ef167059-cef0-12c4-49db-993ca3a20c01,2190959.000000004,1563,High Risk
c03807d7-0b42-0490-23f0-8caeb7c38e74,1597934.100000001,111,High Risk
1f528581-7613-2b96-dce3-68c7cf967ea1,1427488.81,112,Medium Risk
454bbec4-42bc-bd2c-fe3a-f8f4fc4346e9,1241790.09,120,High Risk
10a1709a-5dde-1e3a-4c06-d8de03764712,1211655.7099999997,80,Medium Risk
7ec518e6-787e-cc8a-28f2-6da51fe584b8,1114564.2399999993,55,Medium Risk
da97668c-bf70-6c4d-3157-24aae70ed237,1084982.6500000006,138,Medium Risk
7bc66758-5d59-0b38-1532-29a33fa146c1,1064065.9899999993,158,High Risk
